### PR TITLE
[RUNTIME] Add test for Actions.sleep

### PR DESCRIPTION
**Pull request by jiangdaniel**

### CODE REVIEW

File: runtime/testy/studentCode.py [hunk 1]
```diff
@@ -203,3 +204,34 @@ def optionalTestsWork_setup():
...
+  if not sleepTestVal['test']:
+    print('Success!')
+  else: 
+    print('Failed to sleep for the correct amount of time')
+
+async def asyncSleepHelper2(sleepTestVal):
+  sleepTestVal['test'] = True
+  await Actions.sleep(.5)
+  sleepTestVal['test'] = False
+
```
Comment (kevinji) [inline runtime/testy/studentCode.py]: Really minor thing but I think this is an extra newline.

**Review by yizow — CHANGES_REQUESTED** [review]
Remove one extra new line at end of file, otherwise LGTM

**Comment by kevinji** [comment]
@yizow Just pushed a change that removes the newline, so this should be good to go after Travis finishes.

**Comment by jiangdaniel** [comment]
@mc10 Thanks!

**Review by yizow — APPROVED** [review]

**Pull request merged**